contact.html

```diff
@@ -1,4 +1,63 @@
 <!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Contact</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="css/navbar-style.css" />
+    <link rel="stylesheet" href="css/contact-style.css" />
+  </head>
+
+  <body>
+    <body>
+      <nav class="navbar">
+        <a href="index.html"
+          ><img src="images/Navbar-Logo.png" alt="Navbar Logo" id="navbar-logo"
+        /></a>
+        <a href="#" class="toggle-button" style="margin-top: 20px">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </a>
+        <div class="navbar-links">
+          <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="calendar.html">Calendar</a></li>
+            <li class="dropdown">
+              <a href="committee.html">Committees</a>
+            </li>
+            <li><a href="projects.html">Projects</a></li>
+            <li><a href="merch.html">Merch</a></li>
+            <li style="margin-right: 10px">
+              <a href="contact.html">Contact</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+
+      <script src="js/mobile-toggle.js" async defer></script>
+    </body>
+    <h1>Either scan the code or hit the button</h1>
```

Comment:
```suggestion
    <h1>Scan the code or hit the button</h1>
```
I think it sounds a little cleaner without the "either"